Example_Detection_FCI.py

```diff
@@ -77,25 +113,63 @@ def main():
+
+        # Here we don't load the land/sea mask as we're cropping and this is
+        # not (yet) supported by pyfires. For full disk processing you will
+        # likely get more accurate results by enabling the land/sea mask.
+        initial_load_data_dict = initial_load(
+            fci_files,          # Input file list
+            'fci_l1c_nc',       # Satpy reader name
+            bdict,              # Band mapping dict
+            do_load_lsm=False,  # Don't load land-sea mask
+            bbox=bbox           # Bounding box for cropping
+        )
+
+        alternative_load_data_dict = alternative_load(
+            fci_files,          # Input file list
+            'fci_l1c_nc',       # Satpy reader name
+            bdict,              # Band mapping dict
+            do_load_lsm=False,  # Don't load land-sea mask
+            bbox=bbox           # Bounding box for cropping
+        )
+
+        
+        # direct load
+        vi1_rad = DataQuery(name=bdict['vi1_band'], calibration="radiance")
+        vi2_rad = DataQuery(name=bdict['vi2_band'], calibration="radiance")
+        mir_rad = DataQuery(name=bdict['mir_band'], calibration="radiance")
+        mir_bt = DataQuery(name=bdict['mir_band'], calibration="brightness_temperature")
+        lwi_rad = DataQuery(name=bdict['lwi_band'], calibration="radiance")
+        lwi_bt = DataQuery(name=bdict['lwi_band'], calibration="brightness_temperature")
+
+        blist = [vi1_rad, vi2_rad, mir_rad, lwi_rad, mir_bt, lwi_bt]
+
+        scn = Scene(fci_files, reader='fci_l1c_nc')
+        scn.load(blist, calibration='radiance', generate=False)
+
         if bbox:
             scn = scn.crop(xy_bbox=bbox)
 
-        # Load the initial data.
-        fci_files = [str(f) for f in input_file_dir.glob("*.nc")]
+        scnr = scn.resample(scn.coarsest_area(), resampler='native')
+        
         # Here we don't load the land/sea mask as we're cropping and this is
         # not (yet) supported by pyfires. For full disk processing you will
         # likely get more accurate results by enabling the land/sea mask.
-        data_dict = initial_load(fci_files,        # Input file list
-                                    'fci_l1c_nc',         # Satpy reader name
-                                    bdict,             # Band mapping dict
-                                    do_load_lsm=False,  # Don't load land-sea mask
-                                    bbox=bbox)         # Bounding box for cropping
-
-        # Set up the constants used during processing
-        data_dict = set_default_values(data_dict)
+        direct_load_data_dict = sort_l1(
+            scnr[vi1_rad],
+            scnr[vi2_rad],
+            scnr[mir_rad],
+            scnr[lwi_rad],
+            scnr[mir_bt],
+            scnr[lwi_bt],
+            bdict,
+            do_load_lsm=False,
+        )
+
+        # # Set up the constants used during processing
+        data_dict = set_default_values(direct_load_data_dict)
```

Comment:
@fwfichtner I add comparison of different ways of loading the FCI data. The `initial_load` is the original method defined in `PYF_basic.py`. The `alternative_load` is exactly the same as `initial_load`. Starting from L140 is the direct loading. 

After testing, I found that the dask arrays can be computed only if we load the data directly with `sort_l1`.